.gitattributes

```diff
@@ -0,0 +1 @@
+/raw_templates.go -diff
```

Comment:
Really not a fan of making things disappear from diffs. `git add -p` misses "binary" files, making it easy to forget to commit them.

Reply:
> Really not a fan of making things disappear from diffs.

But those files _do_ appear in diffs:

```
% git diff
diff --git a/raw_templates.go b/raw_templates.go
index 43ca45a..db7f2ee 100644
Binary files a/raw_templates.go and b/raw_templates.go differ
```

And you can always turn the behaviour off with `--text`.

Try diffing a bunch of changes alongside a change to a `composer.lock` file. You'll have to scroll through pages and pages and pages of garbage before, after, or between the actual changes you're interested in. This makes it harder to figure out what meaningful changes have been made in a commit (both for the writers and readers of the commits).

> `git add -p` misses "binary" files, making it easy to forget to commit them.

There's some principle that's like "code is read more than it's written, so optimise for reading". So while I do see the validity of this complaint, I think we should make reading commits easier even at the cost of making writing a little harder.